framework/src/modules/chain/loader.js

```diff
@@ -99,47 +98,37 @@ class Loader {
 	 */
 	// eslint-disable-next-line class-methods-use-this
 	isActive() {
```

Comment:
Why do we have `isActive()` and `syncing()` that return the same property? I think we should only have `syncing()` that is more clear about what is happening.